exotic/exotic_gui.py

```diff
@@ -67,10 +67,10 @@
 except ImportError:  # package import
     from api.filters import fwhm as photometric_filters
 
-try:  # nea
-    from exotic import NASAExoplanetArchive
-except ImportError:
-    from .exotic import NASAExoplanetArchive
+try:
+    from .api.nea import NASAExoplanetArchive
+except ImportError:  # package import
+    from api.nea import NASAExoplanetArchive
```

Comment:
will this still work if a user installs exotic via pip (i.e., the user never downloads all the files from github)

Reply:
This should work via pip as we import other packages this way as well. To give a few examples: 
```
try:  # plate solution
    from .api.plate_solution import PlateSolution
except ImportError:  # package import
    from api.plate_solution import PlateSolution
try:  # nonlinear limb darkening numerics
    from .api.gael_ld import createldgrid
except ImportError:  # package import
    from api.gael_ld import createldgrid
try:  # filters
    from .api.filters import fwhm
except ImportError:  # package import
    from api.filters import fwhm
try:  # light curve numerics
    from .api.elca import lc_fitter, binner, transit
except ImportError:  # package import
    from api.elca import lc_fitter, binner, transit
```